deploy/examples/multus-validation.yaml

```diff
@@ -0,0 +1,138 @@
+#################################################################################################################
+# We highly recommend validating your Multus configuration before you install Rook.
+# This job aims to automate that operation by using the validation tool. Run this job after
+# installing the rook operator and before installing any Custom Resources.
+#################################################################################################################
+# ---
+# scc for the Rook and Ceph daemons
+# kind: SecurityContextConstraints
+# apiVersion: security.openshift.io/v1
+# metadata:
+#   name: rook-ceph-multus-validation
+# allowPrivilegedContainer: true
+# allowHostDirVolumePlugin: true
+# allowHostPID: false
+# # set to true if running rook with host networking enabled
+# allowHostNetwork: true
+# # set to true if running rook with the provider as host
+# allowHostPorts: true
+# priority:
+# allowedCapabilities: ["MKNOD"]
+# allowHostIPC: true
+# readOnlyRootFilesystem: false
+# # drop all default privileges
+# requiredDropCapabilities: ["All"]
+# defaultAddCapabilities: []
+# runAsUser:
+#   type: RunAsAny
+# seLinuxContext:
+#   type: RunAsAny
+# fsGroup:
+#   type: RunAsAny
+# supplementalGroups:
+#   type: RunAsAny
+# seccompProfiles:
+#   - "*"
+# volumes:
+#   - configMap
+#   - emptyDir
+#   - projected
+# users:
+#   - system:serviceaccount:rook-ceph:rook-ceph-multus-validation # serviceaccount:namespace:cluster
+---
+# Aspects of multus validation job that require access to the operator/cluster namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps", "configmaps/finalizers", "pods"]
+    verbs: ["get", "list", "create", "update", "delete"]
+  - apiGroups: ["apps"]
+    resources: ["daemonsets"]
+    verbs: ["list", "create", "delete", "deletecollection"]
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources: ["network-attachment-definitions"]
+    verbs: ["get"]
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["get", "list", "delete"]
+---
+# Allow the multus validation job to run in this namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: rook-ceph-multus-validation
+subjects:
+  - kind: ServiceAccount
+    name: rook-ceph-multus-validation
+    namespace: rook-ceph # namespace:cluster
+---
+# Service account for job that validates multus configuration
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-ceph-multus-validation
+  namespace: rook-ceph # namespace:cluster
+# imagePullSecrets:
+#   - name: my-registry-secret
+---
+# A job that runs the multus validation tool
+apiVersion: batch/v1
+kind: Job
```

Comment:
Because this is the primary "thing" that users will be modifying, make sure this is the first yaml definition in this manifest file.

Reply:
That's unfortunate, but I think you're right to leave where it is then. Users generally report errors like this as bugs. As long as the comment at the top of the file is the primary documentation place, users still can see the primary docs early in the file, which is good.